src/components/loading-component/loading-component.module.css

```diff
@@ -0,0 +1,53 @@
+#mainContainer {
+  height: 30vh;
+  transform: rotateX(80deg) rotateY(25deg);
+  transform-style: preserve-3d;
+}
+
+#centralStar {
+  top: 40%;
+  box-shadow: 0 0 60px rgba(244, 244, 245, 0.3);
+  transform: rotateX(-80deg) translateX(-50%);
+}
+
+#orbit {
+  margin-top: -7.5rem;
+  margin-left: -7.5rem;
+  border: 2px solid rgba(244, 244, 245, 0.2);
+  transform-style: preserve-3d;
+  animation-name: orbit;
+}
+
+#planet {
+  border-width: 0.1px;
```

Comment:
You can use the tailwinds class `border` to get `1px` border width.

Reply:
But then again, if you prefer to leave everything here, feel free to do so. I guess that if we wanted to be really optimised, we should add all the missing css rules in our tailwind theme. I guess this would be overkill.

Reply:
>You can use the tailwinds class border to get 1px border width.

`1px` is way too wide for the wanted result, and unfortunately `0.01px` value isn't in TW [default values](https://tailwindcss.com/docs/border-width).

Reply:
I read it wrong, my bad. I thought it was `0.1rem` 🙇‍♂️.